client/lib/react-player/bower.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
There are four dependencies: react, react-dom, react-player, and underscore. React-player does not exist as a CDN because it's not a hugely popular library, the other three are imported in index.html with CDNs